### DISCUSSION TITLE
Capsicain just spamming error when started

### DISCUSSION THREAD

**Discussion by Deaquay** [cajhin/capsicain#71] [Q&A] [answered]
When i start capsicain with a simple code like:
```
[CONFIG_1]
REWIRE A Z
```
or even with an empty .ini, it just spams this in the main window
```
ERROR: Received unexpected SC_NOP Key Stroke code 0. Ignoring this:

new keyboard: IBM keyboard
```
over and over again. So fast you cant even read what the message says if you dont stop the window from scrolling.
Any fixes?

**Comment by Maickeli** [comment, marked as answer]
I had that error when I tried to run capsicain without installing the Interception driver. Details for installing it are in the installation guide: [Installation](https://github.com/cajhin/capsicain/wiki/Installation)

I haven't done any research where the error comes but I guess that's one reason it may throw that error.
